src/lib/components/Cytoscape.react.js

```diff
@@ -633,7 +633,7 @@ Cytoscape.propTypes = {
             'grid',
             'breadthfirst',
             'cose',
-            'close-bilkent',
+            'cose-bilkent',
```

Comment:
is that the only line you modified before running `npm run build`? just making sure :)

Reply:
yes and in a few other places that were mostly documentation. I tested it by setting the layout to `cose-bilkent` and it works.